examples/3d/3d_scene.rs

```diff
@@ -1,10 +1,14 @@
 //! A simple 3D scene with light shining over a cube sitting on a plane.
 
+use bevy::diagnostic::LogDiagnosticsPlugin;
 use bevy::prelude::*;
+use bevy::render::diagnostic::RenderDiagnosticsPlugin;
 
 fn main() {
     App::new()
         .add_plugins(DefaultPlugins)
+        .add_plugins(RenderDiagnosticsPlugin)
```

Comment:
Probably should remove this before merging.

Reply:
```suggestion
```